sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -358,50 +358,180 @@ class StatisticsSuite extends QueryTest with TestHiveSingleton with SQLTestUtils
     }
   }
 
-  test("generate column-level statistics and load them from hive metastore") {
+  test("test refreshing table stats of cached data source table by `ANALYZE TABLE` statement") {
```

Comment:
Could you deduplicate the two test cases `refreshing table stats` and `refreshing column stats` by calling the same common function?